### PR TITLE
Update GMT::Bsmap::MethRatio

### PR DESCRIPTION
**Pull request by iferguson90**
Supersedes #244.

- [x] Pull test data from Bsmap
- [x] Update tests to run `methratio.py`
- [x] Verify that removing default version doesn't break anything

### CODE REVIEW

**Comment by chrisamiller** [comment]
In response to our backchannel messages, I can't see any reason why removing the default version would break things. +1

File: lib/perl/Genome/Model/Tools/Bsmap/MethRatio.t [hunk 1]
```diff
@@ -4,10 +4,90 @@ use strict;
...
+
+        my $fasta = Genome::Model::Build::ReferenceSequence->get(
+            name => $mapping{$ref_short_name}
+        )->cached_full_consensus_path('fa');
+
+        is($test_obj->_reference_fasta, $fasta, 'Got correct fasta object');
+    }
+}
+
+{
```
Comment (tmooney) [inline lib/perl/Genome/Model/Tools/Bsmap/MethRatio.t]: Should these blocks be [subtests](http://search.cpan.org/~exodist/Test-Simple-1.001014/lib/Test/More.pm#subtest)?

**Comment by acoffman** [comment]
+1 :shipit:

**Comment by tmooney** [comment]
+1 :pineapple:

**Pull request merged**